scripts/zones/Ship_bound_for_Mhaura_Pirates/IDs.lua

```diff
@@ -18,20 +18,158 @@ zones[xi.zone.SHIP_BOUND_FOR_MHAURA_PIRATES] =
         LOGIN_CAMPAIGN_UNDERWAY = 7001, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER            = 7002, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         FISHING_MESSAGE_OFFSET  = 7231, -- You can't fish here.
-        ON_WAY_TO_MHAURA        = 7332, -- We're on our way to Mhaura. We should be there in [less than an hour/about 1 hour/about 2 hours/about 3 hours/about 4 hours/about 5 hours/about 6 hours/about 7 hours] (<number> [minute/minutes] in Earth time).
+        ON_WAY_TO_MHAURA        = 7339, -- We're on our way to Mhaura. We should be there in [less than an hour/about 1 hour/about 2 hours/about 3 hours/about 4 hours/about 5 hours/about 6 hours/about 7 hours] (<number> [minute/minutes] in Earth time).
         LOKHONG_SHOP_DIALOG     = 7337, -- There's nothing like fishing to pass the time!
         CHHAYA_SHOP_DIALOG      = 7338, -- May I offer you items to help you on your journey?
-        ARRIVING_SOON_MHAURA    = 7339, -- We are on our way to Mhaura. We will be arriving soon.
+        ARRIVING_SOON_MHAURA    = 7346, -- We are on our way to Mhaura. We will be arriving soon.
     },
     mob =
     {
-        WIGHT      = 17711120,
-        SILVERHOOK = 17711121,
-        SEA_MONK   = 17682442,
-        SEA_HORROR = 17682446,
+        SEA_CREATURES =
+        {
+            17711110,   -- Sea Crab 1
+            17711111,   -- Sea Crab 2
+            17711112,   -- Sea Pugil 1
+            17711113,   -- Sea Pugil 2
+            17711114,   -- Sea Monk 1
+        },
+        PHANTOM = 17711115,
```

Comment:
Put the ID in {}.

Reply:
Also not needed? Phantom isn't controlled normally and is spawned by hour checks in zones. No reasons to have it put in the normal mob spawn groups for sea_creatures.